src/pk-main.c

```diff
@@ -165,13 +165,13 @@ main (int argc, char *argv[])
 	conf = g_key_file_new ();
 	conf_filename = pk_util_get_config_filename ();
 	if (conf_filename == NULL) {
-		g_printerr ("Config file was not found.\n");
+		g_printerr (_("Config file was not found.\n"));
```

Comment:
I don't think the translators like newlines; e.g. I think you want:

g_printerr ("%s\n", _("Config file was not found."));